checkout-service/.gitignore

```diff
@@ -0,0 +1 @@
+deploy.sh
```

Comment:
What is this file for ?

Reply:
It's a bash script file. Like a command you would type in the terminal. I'm using a aws CLI command to upload my updated bundle with an npm command.

Reply:
Got it! That's Great!